### PR TITLE
LIVE-2566: Prepare for square ads

### PR DESCRIPTION
**Pull request by faresite**
## Why are you doing this?

This PR prepares for square ads format once enabled. Only for first ad in an article.

## Changes

- prepare a boolean flag to turn on square ads
- `.ad-slot-square` styles.

## Screenshots

| Before | After |
| --- | --- |
|  |  |

### CODE REVIEW

**Review by abeddow91 — APPROVED** [review]
LGTM!

File: src/client/nativeCommunication.ts [hunk 1]
```diff
@@ -15,6 +15,8 @@ import {
 	videoClient,
 } from '../native/nativeApi';
 
+const TEADS_ENABLED = true;
```
Comment (JamieB-gu) [inline src/client/nativeCommunication.ts:18]: Could you elaborate on what this flag is for?

**Review by JamieB-gu — APPROVED** [review]

File: src/client/nativeCommunication.ts (hunk 1 above)
Comment (faresite) [reply src/client/nativeCommunication.ts:18]: Actually it should be `false` for now. Glad you mentioned it. It's more for other developers in the future if they need to turn on or off the square slots for teads, they just do it using this flag as opposed to by changing the code logic itself.

Comment (JamieB-gu) [reply src/client/nativeCommunication.ts]: Do you think we could pull that upstream into `mobile-static`/Firebase so that it can be configured remotely?

Comment (faresite) [reply src/client/nativeCommunication.ts]: I thought about it and still not sure what's the best to do. The reason I went with this approach after all:
a) less complicated
b) it doesn't rely on making a call to an external service to decide if it should render the square slot or not. Which could be quite important when we're serving ads.

What are your thoughts @JamieB-gu ?

Comment (JamieB-gu) [reply src/client/nativeCommunication.ts]: I think this is reasonable while we're rolling out Teads. But I don't think we necessarily need this flag here after we've turned it on and Teads are in production.

I think if we reckon there's a reasonable chance we may want to turn these off sometimes then we should have a switch in remote config. If we _don't_ think we'll turn them off sometimes then I don't think we need a flag/switch in general.

Comment (faresite) [reply src/client/nativeCommunication.ts]: In my tenure here we've had to turn on/off different 3rd party providers (including Teads) based on product requirements etc.. The problem with remote switch is we are depending on an external service, but maybe that's not too bad when we're having caching. 
I'm for introducing this as it is to unblock native devs and work on the remote switch at a later date?

**Pull request merged**